lib/generators/install/templates/cable.yml.erb

```diff
@@ -0,0 +1,14 @@
+# production: &production
+#   :url: redis://localhost:6379
+#   :host: localhost
+#   :port: 6379
+#   :timeout: 1
+
+development: &development
+  :url: redis://localhost:6379
+  :host: localhost
+  :port: 6379
+  :timeout: 1
+  :inline: true

```

Comment:
@kayakyakr nice point, @TheAustinSeven are you up for amending the README here too? :grin: